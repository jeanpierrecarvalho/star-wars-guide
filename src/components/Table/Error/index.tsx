import React from "react";

interface IProps {
  error: string;
}

const Error: React.FC<IProps> = ({ error }) => {
  return <p className="mt-8 text-center text-red-500">{error}</p>;
};

export default Error;
