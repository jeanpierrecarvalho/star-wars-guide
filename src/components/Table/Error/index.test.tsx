import { render } from '@testing-library/react'
import { describe, it, expect } from 'vitest'

import Error from '@/components/Table/Error'

describe('Error Component', () => {
  it('matches the snapshot', () => {
    const props = {
      error: 'Lorem Ipsum',
    }

    const { container } = render(<Error {...props} />)
    expect(container).toMatchSnapshot()
  })
})
