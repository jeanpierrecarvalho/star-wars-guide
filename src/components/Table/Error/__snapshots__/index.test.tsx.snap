// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Error Component > matches the snapshot 1`] = `
<div>
  <p
    class="mt-8 text-center text-red-500"
  >
    Lorem Ipsum
  </p>
</div>
`;
